### PR TITLE
Rename FeeWindow to DisputeWindow

### PR DESCRIPTION
**Pull request by nuevoalex**
Just a rename of FeeWindow -> DisputeWindow

### CODE REVIEW

**Comment by MicahZoltu** [comment]
Same argument as before on why DisputeWindow is an inaccurate name that evokes the incorrect mental model in readers.

**Comment by nuevoalex** [comment]
These are used very specifically to space out the time between dispute rounds and to give a window to initially dispute.

Seems like calling them dispute windows is very apt to me.

**Comment by MicahZoltu** [comment]
Hmm, looking at the contents of this contract, it doesn't appear to contain what I thought it contained.  It appears to just be a place for tracking number of invalid/no-show markets?

**Comment by nuevoalex** [comment]
That's all that really gets recorded on it, correct. If anything that is the coincidental data though since we have a non-arbitrary reason for spacing the dispute windows as we do. We could adjust bonds at a different cadence if we wanted but this is a nice way to do it.

File: tests/reporting/test_bond_redemption.py [hunk 1]
```diff
@@ -69,12 +69,12 @@ def test_failed_crowdsourcer(finalize, localFixture, universe, market, cash, rep
 
     if finalize:
         # Fast forward time until the fee window is over and we can redeem to recieve the REP back
```
Comment (adrake33) [inline tests/reporting/test_bond_redemption.py]: Looks like there are still some references to "fee window" as 2 words. Recommend doing a find/replace on those as well.

**Pull request merged**